userspace/falco/webserver.h

```diff
@@ -36,12 +38,8 @@ class falco_webserver
 	falco_webserver& operator = (const falco_webserver&) = delete;
 	virtual void start(
 		const std::shared_ptr<sinsp>& inspector,
-		uint32_t threadiness,
-		uint32_t listen_port,
-		std::string& list_address,
-		std::string& healthz_endpoint,
-		std::string &ssl_certificate,
-		bool ssl_enabled);
+		const falco_configuration::webserver_config& configuration,
```

Comment:
Sounding like a broken record also here it would be better to name it very specific `webserver_config` instead of just a generic `configuration`.

Reply:
No problem with such records :-)

Done